testsuite/tests/unwind/Makefile

```diff
@@ -0,0 +1,19 @@
+default:
+	@if $(BYTECODE_ONLY); then : ; elif [ $(SYSTEM) = macosx ]; then $(MAKE) native_macosx_tests; fi
+
+native_macosx_tests:
+	@printf " ... testing 'unwind_test':"; \
+        $(MAKE) clean ; $(MAKE) unwind_test && \
+	./unwind_test >/dev/null 2>&1 && echo " => passed" || echo " => failed"
+
+unwind_test:
+	@ocamlopt -c -opaque mylib.mli
```

Comment:
(well, [opam-compiler-conf](https://github.com/gasche/opam-compiler-conf) makes it very easy to install a switch for the compiler branch you are currently hacking on.)